django_x509/base/admin.py

```diff
@@ -99,7 +99,7 @@ class AbstractCaAdmin(BaseAdmin):
               'modified']
 
     class Media:
-        js = ('django-x509/js/x509-admin.js',)
+        js = ('admin/js/jquery.init.js', 'django-x509/js/x509-admin.js',)
```

Comment:
Can you move it to the next line? 
(Curtesy: pep8 -- read about it if you haven't already! :smile:)

Reply:
Yeah, done